test/external/external_test_whisper_librispeech.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+import torch
+import tqdm
+import torchaudio
+import pathlib
+import jiwer
+import os
+import numpy as np
+from whisper.normalizers import EnglishTextNormalizer
```

Comment:
Figured it'd be easier to just import this rather than copy everything in https://github.com/openai/whisper/tree/main/whisper/normalizers